waiter/project.clj

```diff
@@ -115,7 +115,16 @@
              :test {:jvm-opts
                     [~(str "-Dwaiter.test.kitchen.cmd=" (or
                                                           (System/getenv "WAITER_TEST_KITCHEN_CMD")
-                                                          (.getCanonicalPath (clojure.java.io/file "../kitchen/bin/run.sh"))))]}
+                                                          (.getCanonicalPath (clojure.java.io/file "../kitchen/bin/run.sh"))))]
+                    ;; Print exception data
+                    :injections [(defmethod clojure.test/report :error [m]
```

Comment:
We've already got some `clojure.test/report` `defmethod`s in `test_helpers.clj`. Should we move this one there too?